draft-ietf-tls-tls13.md

```diff
@@ -5278,18 +5278,24 @@ The registries and their allocation policies are below:
 
 - TLS Alert Registry: Future values are allocated via Standards
   Action {{RFC8126}}. IANA \[SHALL update/has updated] this registry
-  to include values for "missing_extension" and "certificate_required".
+  to include values for "missing_extension" and "certificate_required".  The
+  "DTLS-OK" column is marked as "Yes" for each new alert.
 
 - TLS HandshakeType Registry: Future values are allocated via
   Standards Action {{RFC8126}}. IANA \[SHALL update/has updated] this registry
   to rename item 4 from "NewSessionTicket" to "new_session_ticket"
   and to add the "hello_retry_request_RESERVED", "encrypted_extensions",
-  "end_of_early_data", "key_update", and "message_hash" values.
+  "end_of_early_data", "key_update", and "message_hash" values.  The "DTLS-OK"
+  are marked as "Yes" for each of these additions.
 
 This document also uses the TLS ExtensionType Registry originally created in
 {{RFC4366}}. IANA has updated it to reference this document. The registry and
 its allocation policy is listed below:
 
+ - IANA \[SHALL update/has updated] Values with the first byte in the range
```

Comment:
Is this shall/has spurious so that we should just start with "Values"?

Reply:
No I think it's just a bad cut/paste job.  I'll fix shortly.